docs/manuals/core/concepts/index.md

```diff
@@ -43,30 +43,33 @@ This section aims at defining the following Taipy Core concepts.
 - A [Task](task.md) (the orange boxes) can be seen as a function receiving data node(s) as input and returning
   data node(s) as output.
 - A [Job](job.md) represents a unique execution of a Task.
-- A [Pipeline](pipeline.md) represents a set of tasks connected together through data nodes, that should be executed
-  as a whole and forming a consistent algorithm.
-- A [Scenario](scenario.md) is made of one or multiple pipelines. It represents an instance of a business problem to
-  solve.
+- A [Scenario](scenario.md) represents a set of tasks connected together through data nodes forming a Directed Acyclic Graph,
+  that should be executed as a whole (or as subgraphs via the use of sequences) and forming a consistent algorithm. It can
+  also contain a set of additional data nodes for data that are related but are not part of the executable graph.
```

Comment:
```suggestion
- A [Scenario](scenario.md) represents a set of tasks connected through data nodes forming a Directed Acyclic Graph
  that should be executed as a whole (or as subgraphs via sequences) and create a consistent algorithm. It can
  also contain a set of additional data nodes for related data that are not part of the executable graph.
```